tools/api-generator/commands.go.tmpl

```diff
@@ -12,9 +13,10 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-//
-// This code was autogenerated using `make gen-L1-commands`
+
+// Code generated using `make gen-api-commands`. DO NOT EDIT.
```

Comment:
GH will consider any file with `// Code generated ...` as auto generated
Intellij will show a warning when opening files with `DO NOT EDIT.`